pkg/exporter/sumologicexporter/source_format.go

```diff
@@ -80,7 +80,7 @@ func (s *sourceFormat) format(f fields) string {
 		if ok {
 			labels = append(labels, tracetranslator.AttributeValueToString(v))
 		} else {
-			labels = append(labels, "")
+			labels = append(labels, "undefined")
```

Comment:
Make it a `const`?

Reply:
Sure. Does this look OK? https://github.com/SumoLogic/sumologic-otel-collector/pull/88/commits/a7d4d6ae12ab5ff79b842b7f99be220d6497388b

Reply:
Oh I guess I need to use this const in translateConfigValue function too...

Reply:
How about this? https://github.com/SumoLogic/sumologic-otel-collector/pull/88/commits/ad126f294ef470044c2ae627374ce7bc5e9149ff

Reply:
`unrecognizedAttributeValue`?

Reply:
Done